ckanext/unaids/theme/templates/request/show.html

```diff
@@ -0,0 +1,83 @@
+{% extends "request/page.html" %}
+
+{% block subtitle %}{{ _('Organization membership request') }}{% endblock %}
+
+{% block secondary_content %}
```

Comment:
are you introducing any changes in the `subtitle` and `secondary_content` blocks? If not no need to add them here. Only the `primary_content` block includes changes to the original ext.